apps/src/lab2/views/components/Instructions/ResourcePanel/styles.module.scss

```diff
@@ -130,5 +130,6 @@
 
 .navigationFooter {
   padding: 10px;
-  background-color: var(--background-neutral-secondary)
+  background-color: var(--background-neutral-secondary);
+  border-top: variables.$default-border;
```

Comment:
I was about to submit a follow-up for missing this in https://github.com/code-dot-org/code-dot-org/pull/68541 but you already caught it 😄 thanks Kelby!